pkg/proctree/proctree_procfs.go

```diff
@@ -308,38 +308,43 @@ func dealWithThread(pt *ProcessTree, givenPid, givenTid int32) error {
 
 // dealWithProcFsEntry deals with a process from procfs.
 func dealWithProcFsEntry(pt *ProcessTree, givenPid int32) error {
-	_, err := os.Stat(fmt.Sprintf("/proc/%v", givenPid))
-	if os.IsNotExist(err) {
-		return errfmt.Errorf("could not find process %v", givenPid)
-	}
-
-	err = dealWithProc(pt, givenPid) // run for the given process
+	err := dealWithProc(pt, givenPid) // run for the given process
 	if err != nil {
+		if os.IsNotExist(err) {
+			err = nil // ignore non-existent processes
+		}
 		if debugMsgs {
 			logger.Debugw("dealWithProc", "pid", givenPid, "err", err)
 		}
+
 		return err
 	}
 
-	taskPath := fmt.Sprintf("/proc/%v/task", givenPid)
+	taskPath := "/proc/" + strconv.FormatInt(int64(givenPid), 10) + "/task"
```

Comment:
Shouldn't we add `getTaskPath` in formatters?

Reply:
I believe I didn't created it since it's the unique use. But good idea, let me bring it.